docs/src/components/Hero/SelectedLabel/index.tsx

```diff
@@ -88,12 +88,7 @@ const SelectedLabel: React.FC<{
     );
   };
 
-  const classList = clsx(
-    isInteractive ? styles.preEnabledTextInteractivity : null,
-    staticStyles.current.enabledTextInteractivity
-      ? styles.interactiveHeaderText
-      : styles.headerText
-  );
+  const classList = styles.headerText;
```

Comment:
You don't need variable here `const classList`, you can just write className={styles.headerText}